### PR TITLE
Roadmap

### PR DESCRIPTION
**Pull request by dinagraves**
First draft

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Mission and Vision
+
+Four Keys mission:
+
+  Be the industry-standard reference implementation for the automated calculation of the DORA Four Keys metrics.  
+
+The vision for this is:
+
+* All discussion of the DORA Four Key metrics center around the key data entities of Changes, Deployments, and Incidents
```
Comment (donmccasland) [inline ROADMAP.md:11]: Grab language from PRD, make sure users understand that the only metrics this project will ever provide are the Four Key metrics.

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+* A rich set of tool integrations and data mappings
```
Comment (donmccasland) [inline ROADMAP.md:12]: And that the rest of the work will be around integrations, different deployment patterns, and ease of use.

**Review by donmccasland — APPROVED** [review]

File: ROADMAP.md (hunk 1 above)
Comment (dinagraves) [reply ROADMAP.md:11]: Added a little section for "Non-Goals"

**Comment by dinagraves** [comment]
Merging this in - feel free to submit addition PR's

**Pull request merged**